apis/v1alpha1/instrumentation_types.go

```diff
@@ -80,6 +80,19 @@ type PythonSpec struct {
 	Image string `json:"image,omitempty"`
 }
 
+// Resource defines resource configuration.
+type Resource struct {
+	// Attributes defines attributes that are added to the resource.
```

Comment:
And here. It might be a good idea to provide an example.

Reply:
any suggestions to naming are welcome